src/testUtils/db.ts

```diff
@@ -36,13 +36,13 @@ export function setUpTestDbConnection(): () => Connection {
   });
 
   beforeEach(async () => {
-    if (connection.readyState === 0) {
+    if (connection.readyState === STATES.disconnected) {
       connection = await connect();
     }
   });
 
   afterEach(async () => {
-    if (connection.readyState === 0) {
+    if (connection.readyState === STATES.disconnected) {
```

Comment:
Irrelevant to this PR, but I need the latest ESLint config here and it requires these changes.